epub33/core/index.html

```diff
@@ -6086,11 +6075,28 @@ <h4>URLs in the OCF Abstract Container</h4>
 						<li>Return <var>false</var>.</li>
 					</ol>
 
-					<p class="note">
-						For better interoperability with non-conforming or legacy Reading Systems and toolchains, 
-						EPUB Creators should not use more <a data-cite="url/#double-dot-path-segment">double-dot path segments</a> 
-						than needed to reach the target container file.
-					</p>							
+					<p>
+						In the <a>OCF Abstract Container</a>, any URL string MUST be an <a data-cite="url#absolute-url-with-fragment-string">absolute-URL-with-fragment-string</a> or a <a>valid-relative-container-URL-string</a>.
```

Comment:
same nitpick here: remove the last hyphen before "string"

Reply:
yes, module the change of the reference to also include "with-fragment"